docs/install.rst

```diff
@@ -366,8 +368,8 @@ can be used to install wasp-bootloader:
 .. note::
 
     If you want to restore the PineTime factory firmware then you can
-    use nRF Connect to do this. Perform a long press reset and then
-    use nRF Connect to send ``reloader-factory.zip`` to the *PineDFU*
+    use Gadgetbridge to do this. Perform a long press reset and then
+    use Gadgetbridge to send ``reloader-factory.zip`` to the *PineDFU*
```

Comment:
maybe inform the user about the unsupported device thing ("see note in Installing wasp-os section" or something)